apps/fluent-tester/src/FluentTester/TestComponents/Svg/SvgTest.tsx

```diff
@@ -118,6 +119,9 @@ const BundledSvgTest: React.FunctionComponent = () => {
 };
 
 const RemoteSvgTest: React.FunctionComponent = () => {
+  // Temporarily stop testing this case until it can be either more robust or removed
```

Comment:
Add an issue tag?

Reply:
I was going to add an issue tag & merge this once I got a response from warren about what should be done about this test

Reply:
Added the issue tag.